convert_tendl_rand.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import os
+import sys
+import tarfile
+
+from multiprocessing import Pool
+
+import openmc.data
+from utils import download
+
+description = '''
+Download random TENDL libraries from PSI and convert it to a HDF5 library for use with OpenMC. 
+Only certain nuclides are available from PSI. This script generates a cross_sections_Tendl.xml 
+file with random TENDL evaluations plus a standard library located in 'OPENMC_CROSS_SECTIONS'
+'''
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+
+#n_choices=['all', 'O16', 'Na23', 'Si28', 'Si29','Si30', 'Fe054', 'Fe056', 'Fe057', 'Fe058', 'Ge','Pu240',]
+n_choices=['all', 'O16','Si28', 'Si29','Si30', 'Fe54', 'Fe56', 'Fe57', 'Fe58']
```

Comment:
Generally, we like to stay consistent with style by following the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guide. If you're too lazy to think about what the rules are, I'd recommend using a program like [black](https://pypi.org/project/black/) to autoformat your code.